service/view_building_coordinator.cc

```diff
@@ -196,4 +196,4 @@ future<std::vector<canonical_mutation>> view_building_coordinator::remove_view(c
 
 }
 
-}
\ No newline at end of file
+}
```

Comment:
Nit: please make sure to handle the endline at the end of file in the right commit - right now it is done in `view_building_worker::consumer`, but should be done in the commit which introduces the VBC.